src/GitHub.Exports/Services/GitService.cs

```diff
@@ -69,37 +49,23 @@ public static UriString GetUriFromPath(string path)
         /// </remarks>
         /// <param name="path">The path to start probing</param>
         /// <returns>An instance of <see cref="IRepository"/> or null</returns>
-        public IRepository GetRepo(string path)
+        public IRepository GetRepository(string path)
         {
             var repoPath = Repository.Discover(path);
             return repoPath == null ? null : new Repository(repoPath);
         }
 
         /// <summary>
-        /// Probes for a git repository and if one is found, returns a <see cref="IRepository"/> instance for the
-        /// repository.
-        /// </summary>
-        /// <remarks>
-        /// The lookup checks to see if the specified <paramref name="path"/> is a repository. If it's not, it then
-        /// walks up the parent directories until it either finds a repository, or reaches the root disk.
-        /// </remarks>
-        /// <param name="path">The path to start probing</param>
-        /// <returns>An instance of <see cref="IRepository"/> or null</returns>
-        public static IRepository GetRepoFromPath(string path)
-        {
-            return GitServiceHelper.GetRepo(path);
-        }
-
-        /// <summary>
         /// Returns a <see cref="UriString"/> representing the uri of the "origin" remote with no modifications.
         /// </summary>
         /// <param name="repo"></param>
+        /// <param name="remote">The name of the remote to look for</param>
         /// <returns></returns>
-        public static UriString GetOriginUri(IRepository repo)
+        public UriString GetOriginUri(IRepository repo, string remote = "origin")
```

Comment:
This probably should now be called "GetRemoteUri" and shouldn't have "origin" in its doc comments any more.